script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_4.txt

```diff
@@ -39,27 +39,27 @@ There are a few uncertainties in this strategy.
 <Page12>
 For one, there is the possibility that Vlov will notice me hiding.
 Although Arcueid said that the possibility of this happening is extremely low, in the end, it's still not zero.
-If that happens, there is no guaranteeing my safety. There is a risk that I will be burned without being able to do anything, just like at the hotel.
+If that happens, there is no guaranteeing my safety. There is a risk that I will be burned without being able to do anything, just like the people in the hotel.
 <Page13>
 But that's okay.
 That's a risk I'm willing to take.
 However, there's another problem.
 That is, if Arcueid is unable to restrain Vlov... In other words, the case where Vlov takes her down.
 <Page14>
-...In that scenario, there would be no one else but me.
-It will be up to this single knife to both stop Vlov and save Arcueid.
+...In that scenario, I'd be the only one left.
+It would be up to this single knife to both stop Vlov and save Arcueid.
```

Comment:
Hmm, if you drop the `would` / hypothetical qualifier from the sentence it feels a bit off to me when viewed in conjunction with the sentence above. Would this work for you?

> Stopping Vlov and saving Arcueid; it would all depend on this [single?] knife.

Reply:
not strictly in the Japanese, but I don't see any harm if you think it makes it clearer.